sdk/messaging/azservicebus/receiver.go

```diff
@@ -358,6 +364,83 @@ func (r *Receiver) DeadLetterMessage(ctx context.Context, message *ReceivedMessa
 	return r.settler.DeadLetterMessage(ctx, message, options)
 }
 
+// DeleteMessagesOptions contains the optional parameters for the [Client.DeleteMessages] method.
+type DeleteMessagesOptions struct {
+	// Count is the maximum number of messages to delete.
+	// Defaults to 4000.
+	Count int
```

Comment:
Is there a max value for this that needs to be doc'ed?

Reply:
Currently it's 4000. However, do we normally document service-side limits on the client? It feels similar to client-side validation (ie, we should let the server control the messaging in it's error).

Reply:
Good point. Maybe a link to the docs for this?

Reply:
I added it as a task to the overall DeleteMessages issue (which I had no filed)
https://github.com/Azure/azure-sdk-for-go/issues/22800